mypy/nodes.py

```diff
@@ -2267,6 +2268,7 @@ class SymbolTableNode:
     mod_id = ''  # type: Optional[str]
     # If this not None, override the type of the 'node' attribute.
     type_override = None  # type: Optional[mypy.types.Type]
+    alias_tvars = None  # type: Optional[List[str]]
```

Comment:
It looks like these are qualified names. I added a test for this, and a comment here.